src/components/Annotation/Card/BaseCard.tsx

```diff
@@ -9,64 +9,69 @@ import { TagsWidget } from '../TagsWidget';
 import type { CardProps } from './CardProps';
 
 import './BaseCard.css';
+import type { Translations } from 'src/Types';
 
 type BaseCardProps = CardProps & {
-
   comment(props: CommentProps): ReactNode;
 
-}
+  i18n: Translations;
```

Comment:
Did your IDE complain about this. Shouldn't be necessary, since translations are already included in the `CardProps` base type.